columnflow/production/cms/electron.py

```diff
@@ -168,8 +168,7 @@ def electron_weights_setup(
 
 @electron_weights.init
 def electron_weights_init(self: Producer, **kwargs) -> None:
-    weight_name = self.weight_name
-    self.produces |= {weight_name, f"{weight_name}_up", f"{weight_name}_down"}
+    self.produces.add(f"{self.weight_name}{{,_up,_down}}")
```

Comment:
I think it's great that we can add variables with this notation but it is not very verbose (especially the comma to indicate an empy string). Maybe we want to add comments to clarify that we add three columns to the uses here (nominal+up/down)?

Reply:
Indeed. I think it takes some time to get used to it. I added comments though to make life easier.